packages/app/src/debug/DebugPageDetails.vue

```diff
@@ -24,9 +24,11 @@
       :ci="ci"
     />
     <DebugOverLimit
-      v-else-if="status === 'OVERLIMIT' || isHiddenByUsageLimits"
+      v-else-if="status === 'OVERLIMIT' || isHidden"
+      :over-limit-reason="reasonsRunIsHidden[0] || null"
```

Comment:
Shoot, meant to refactor this and lost track of it. Will look at pushing this down into the lower component.

Side note: Meant to touch base to see if there was a "preferred" reason to display since theoretically the viewed run could be too old *and* newer runs could have pushed them over their plan limit. In that case, we could get *both* causes back - do we care about presenting "usage" over "retention" or vice-versa?